Source/WebCore/html/track/TextTrackCueGeneric.cpp

```diff
@@ -153,7 +153,7 @@ void TextTrackCueGenericBoxElement::applyCSSProperties()
         setInlineStyleProperty(CSSPropertyBackgroundColor, serializationForHTML(cue->backgroundColor()));
     setInlineStyleProperty(CSSPropertyWritingMode, cue->getCSSWritingMode(), false);
     setInlineStyleProperty(CSSPropertyWhiteSpaceCollapse, CSSValuePreserve);
-    setInlineStyleProperty(CSSPropertyTextWrap, CSSValueWrap);
+    setInlineStyleProperty(CSSPropertyTextWrapMode, CSSValueWrap);
```

Comment:
Just confirming that we do not need to also set `text-wrap-style` here?

Reply:
Same here